common/src/main/java/org/mcaccess/minecraftaccess/config/config_maps/MouseSimulationConfigMap.java

```diff
@@ -15,8 +15,8 @@ public class MouseSimulationConfigMap {
     private boolean enabled;
     @SerializedName("Scroll Delay (in milliseconds)")
     private int scrollDelayInMilliseconds;
-    @SerializedName("mac mouse fix")
-	private Boolean macMouseFix;
+    @SerializedName("Mac Mouse Fix")
+	private Boolean macMouseFix = false;
```

Comment:
This line is indented with a tab, it should be indented with four spaces.
```suggestion
    private Boolean macMouseFix = false;
```